tests/baselines/reference/moduleExportDuplicateAlias.errors.txt

```diff
@@ -0,0 +1,16 @@
+tests/cases/conformance/salsa/moduleExportAliasDuplicateAlias.js(1,9): error TS2300: Duplicate identifier 'apply'.
+tests/cases/conformance/salsa/moduleExportAliasDuplicateAlias.js(3,1): error TS2322: Type '() => void' is not assignable to type 'undefined'.
+tests/cases/conformance/salsa/moduleExportAliasDuplicateAlias.js(3,9): error TS2300: Duplicate identifier 'apply'.
+
+
+==== tests/cases/conformance/salsa/moduleExportAliasDuplicateAlias.js (3 errors) ====
+    exports.apply = undefined;
+            ~~~~~
+!!! error TS2300: Duplicate identifier 'apply'.
+    function a() { }
+    exports.apply = a;
+    ~~~~~~~~~~~~~
+!!! error TS2322: Type '() => void' is not assignable to type 'undefined'.
+            ~~~~~
+!!! error TS2300: Duplicate identifier 'apply'.

```

Comment:
To get rid of this error, we can defer detection of this case in the binder by removing the offending case from `AliasExcludes` (like we did for class/function merges) and then look for it in the _checker_, where we can filter certain declarations (probably undefined assignments) and except it, while retaining the error more generally.

I think this pattern is actually _super common_, considering it's our current emit for cjs, so we should probably invest the effort into making it work smoothly.

Reply:
Per our discussion, I switched this to None for the beta. It turns out that the checker already issues an error in this case, but we can add an exclusion for `undefined`-as-initialiser later.